File: 01-felix.py
from pathlib import Path

numbers = [
    sum(map(int, x.split("\n")))
    for x in Path("inputs/01-felix").read_text().split("\n\n")
]

print("Part 1:", max(numbers))
print("Part 2:", sum(sorted(numbers, reverse=True)[:3]))

File: 01-felix.clj
(def calories (->>
  (slurp "inputs/01-felix")
  (#(clojure.string/split % #"\n\n"))
  (map (fn [x] (->> x
    (#(clojure.string/split % #"\n"))
    (map parse-long)
    (reduce +))))
  (sort >)))

(println "Part 1:" (reduce max calories))
(println "Part 2:" (->> calories (take 3) (reduce +)))
